schematic/__init__.py

```diff
@@ -98,6 +139,9 @@ def set_up_tracing(session: requests.Session) -> None:
     if tracing_export == "otlp":
         exporter = OTLPSpanExporter(session=session)
         trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(exporter))
+        # Add the custom AttributePropagatingSpanProcessor to propagate attributes
+        attribute_propagator = AttributePropagatingSpanProcessor(["user.id"])
+        trace.get_tracer_provider().add_span_processor(attribute_propagator)
```

Comment:
I think the order of span_processors being added needs to be flipped. It may cause things to happen out of order in certain scenarios otherwise.

Reply:
Got it. Thank you for pointing that out.